desktop_version/lang/pl/strings.xml

```diff
@@ -502,16 +502,16 @@
     <string english="Press {button} to activate terminals" translation="Aktywuj terminale: {button}" explanation="keyboard key (E or ENTER) or controller button glyph is filled in for {button}. max is when displayed. there are 3 next to each other in the ship" max="37"/>
     <string english="Press {button} to interact" translation="Wejdź w interakcję: {button}" explanation="keyboard key (E or ENTER) or controller button glyph is filled in for {button}. max is when displayed" max="37"/>
     <string english="- Press {button} to skip -" translation="- Pomiń: {button} -" explanation="keyboard key (ENTER) or controller button glyph is filled in for {button}. max is when displayed. This prompt is for skipping cutscenes" max="40"/>
-    <string english="Passion for Exploring" translation="Passion for Exploring" explanation="jukebox prompt. song name should probably not be translated" max="37*2"/>
-    <string english="Pushing Onwards" translation="Pushing Onwards" explanation="jukebox prompt. song name should probably not be translated" max="37*2"/>
-    <string english="Positive Force" translation="Positive Force" explanation="jukebox prompt. song name should probably not be translated" max="37*2"/>
-    <string english="Presenting VVVVVV" translation="Presenting VVVVVV" explanation="jukebox prompt. song name should probably not be translated" max="37*2"/>
-    <string english="Potential for Anything" translation="Potential for Anything" explanation="jukebox prompt. song name should probably not be translated" max="37*2"/>
-    <string english="Predestined Fate" translation="Predestined Fate" explanation="jukebox prompt. song name should probably not be translated" max="37*2"/>
-    <string english="Pipe Dream" translation="Pipe Dream" explanation="jukebox prompt. song name should probably not be translated" max="37*2"/>
-    <string english="Popular Potpourri" translation="Popular Potpourri" explanation="jukebox prompt. song name should probably not be translated" max="37*2"/>
-    <string english="Pressure Cooker" translation="Pressure Cooker" explanation="jukebox prompt. song name should probably not be translated" max="37*2"/>
-    <string english="ecroF evitisoP" translation="ecroF evitisoP" explanation="jukebox prompt. song name should probably not be translated" max="37*2"/>
+    <string english="Passion for Exploring" translation="Pasja do Eksploracji|(Passion for Exploring)" explanation="jukebox prompt. song name should probably not be translated" max="37*2"/>
```

Comment:
The song name and subtitles seem to be inconsistent with the Russian translation. Whereas Russian puts English first and the translation in parentheses after, these put the translation first and English in parentheses after.

Reply:
English song name first, translation as subtitle seems ideal to me. we should probably favor the one we already have and change this one i think

Reply:
I think it should be whatever's appropriate in each language - most languages don't translate the song names at all, while some translate them and remove the English ones completely. So maybe for Russian it's appropriate to keep them English but subtitle them, while for Polish and Silesian it's appropriate to put a translated one first and foremost - consistency across languages should not override that.

Maybe that's not the case here though, so let's ask the translator.

Reply:
Feel free to rearrange these as you see fit. (Will also need to be applied to the unlock prompts in cutscenes.xml)

Reply:
Done and force-pushed.